src/pymor/operators/constructions.py

```diff
@@ -1430,3 +1430,81 @@ def apply_adjoint(self, V, mu=None):
     def apply_inverse_adjoint(self, U, mu=None, initial_guess=None, least_squares=False):
         assert U in self.source
         return self.range.from_numpy(U.to_numpy())
+
+
+class BilinearFunctional(Operator):
```

Comment:
Actually, wouldn't people rather call this a 'quadratic' functional, @TiKeil, @pmli?

Reply:
In my opinion, matching the functional name and the keyword would make sense.